### PR TITLE
Multi torrent bug fix

### PR DESCRIPTION
**Pull request by dxingda**
This pull request resolves [#66](https://github.com/webtorrent/webtorrent-cli/issues/66)
Please try for yourself and give any feedback necessary to get this merged. Thanks.

### CODE REVIEW

**Comment by DiegoRBaquero** [comment]
Willing to rebase to current code base?

**Comment by feross** [comment]
@dxingda Can you update this PR to fix the conflicts?

File: bin/cmd.js [hunk 1]
```diff
@@ -833,13 +829,15 @@ function drawTorrent (torrent) {
   }
 }
 
-function torrentDone () {
+function torrentDone (torrent) {
```
Comment (feross) [inline bin/cmd.js:832]: If we need a reference to `torrent` then I think this function should be moved so that it captures it.

File: bin/cmd.js [hunk 2]
```diff
@@ -833,13 +829,15 @@ function drawTorrent (torrent) {
...
   if (argv['on-done']) {
     cp.exec(argv['on-done']).unref()
   }
 
-  if (!playerName && !serving && argv.out && !argv['keep-seeding']) {
+  if (!playerName && !serving && argv.out && !argv['keep-seeding'] && !numTorrents) {
```
Comment (feross) [inline bin/cmd.js:837]: Clearer to check `numTorrents === 0`

File: bin/cmd.js [hunk 3]
```diff
@@ -202,34 +202,28 @@ if (['info', 'create', 'download', 'add', 'seed'].includes(command) && argv._.le
 } else if (command === 'download' || command === 'add') {
   const torrentIds = argv._.slice(1)
 
-  if (torrentIds.length > 1) {
-    handleMultipleInputs(torrentIds)
-  }
+  if (torrentIds.length > 1) handleMultipleInputs(torrentIds)
+  numTorrents = torrentIds.length
```
Comment (feross) [inline bin/cmd.js:206]: Is there any way to avoid duplicating this code? Seems like it's needed for most of these cases

**Review by feross — APPROVED** [review]

**Comment by feross** [comment]
Thank you for updating this PR @alxhotel – it looks good to merge.

**Comment by feross** [comment]
Made the changes I requested in https://github.com/webtorrent/webtorrent-cli/commit/64e63bd1c200fb6868cd851db826ec4b5494ce0e and released as 3.0.8

**Pull request merged**